gsl/span

```diff
@@ -670,6 +670,19 @@ span<typename Ptr::element_type>
 make_span(Ptr& cont)
 { return span<typename Ptr::element_type>(cont); }
 
+template <class T, class = std::enable_if_t<std::is_pod<T>::value>>
+span<const byte, sizeof(T)>
+as_bytes(const T& e) noexcept
+{
+    return{ reinterpret_cast<const byte*>(&e),sizeof(T) };
+}
+
+template <class T, class = std::enable_if_t<std::is_trivially_copyable<T>::value>>
```

Comment:
@LYP951018: I had actually a few use cases for it, when the result is directly passed to a function (like `socket.write(as_bytes(1+2));`) but I'd tend to agree with you, because it is just too easy to miss those kinds of errors (especially during refactoring).